src/modules/user/dto/update-enrolled.dto.ts

```diff
@@ -0,0 +1,35 @@
+import { ApiProperty } from '@nestjs/swagger';
+
+interface video {
+  num: number;
+  timestamp?: Date;
+}
+
+export class UpdateEnrolledDTO {
+  @ApiProperty({
+    type: [Boolean],
+    description: 'The description of the student',
+    default: '',
```

Comment:
Why have you added default = `''` everywhere?

Reply:
Oh, I will fill something here

Reply:
@Devesh21700Kumar  What does default depicts?

Reply:
i've added the defauts now.
i had not completed.
it just depicts an example value of the property nothing else